values/job-defaults/stream-to-nfs-job.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+enableExtraVolume: true
+appParameters:
+  # Files will be created with this interval.
+  checkpointIntervalMs: "60000"
+mainClass: io.pravega.flinktools.StreamToFileJob
+jobManager:
+  volumeMounts:
+    - mountPath: /mnt/examples-sample
```

Comment:
This file should contain job defaults, not examples. Remove the jobManager and taskManager sections.

Reply:
removed from job-defaults and added in samples.